client/src/components/Header/components/universal-nav.tsx

```diff
@@ -40,12 +42,11 @@ const UniversalNav = ({
   });
 
   const search =
```

Comment:
```suggestion
  const search = isLanding(pathname) ? (
```
since we're not using window, we don't have to check it exists.